frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -133,6 +134,11 @@ export function DashboardItem({
             {...longPressProps}
             data-attr={'dashboard-item-' + index}
         >
+            {item.is_sample && (
+                <div className="sample-dasbhoard-overlay">
+                    <Button onClick={() => router.actions.push(link)}>Configure</Button>
+                </div>
+            )}
```

Comment:
It's sort of annoying that this Configure overlay just pops in on hover with no warning. I think it should be clear that this needs configuration _before_ attempting to interact + the overlay should be less surprising and intrusive (also, something like a 200ms transition would do a lot here).

Reply:
Yup agreed, just updated, wdyt? the delay would be great, but I prefer to keep this in CSS to simplify